docker-compose.yml

```diff
@@ -429,10 +415,126 @@ services:
       - 127.0.0.1:8093:8080
     networks:
       placeos:
-        ipv4_address: 172.31.231.19
     volumes:
       - type: volume
         source: rethink-data
         target: /data/rethinkdb_data
     environment:
       TZ: $TZ
+
+  # Aggregates logs and forwards them to Elasticsearch.
+  logstash:
+    image: docker.elastic.co/logstash/logstash-oss:${ELASTIC_VERSION:-7.6}
+    restart: always
+    container_name: logstash
+    hostname: logstash
+    expose:
+      - 12201/udp
+      - 5044
+    ports:
+      - "127.0.0.1:12201:12201/udp"
+    networks:
+      placeos:
+    volumes:
+      - ./config/logstash/config:/config
+      - ./config/logstash/patterns:/opt/logstash/extra_patterns
+    restart: always
+    labels:
+      container_group: logging
+    logging:
+      driver: json-file
+      options:
+        max-size: 99m
+        labels: container_group
```

Comment:
What's happening here?

Reply:
This adds a field `container_group` to the logs, which at the moment is only set on the new logging services.
It made more sense in the old monitoring setup with more containers, I'll remove it.